lms/static/scripts/frontend_apps/utils/test/onedrive-picker-client-test.js

```diff
@@ -116,25 +116,28 @@ describe('OneDrivePickerClient', () => {
     });
   });
 
-  describe('#encodeSharingURL', () => {
+  describe('.encodeSharingURL', () => {
     [
       {
-        sharingURL: 'https://1drv.ms/b/s!AmH',
+        sharingURL: atob('aHR0cHM6Ly8xZHJ2Lm1zL2IvaQ=='),
         expectedResult:
-          'https://api.onedrive.com/v1.0/shares/u!aHR0cHM6Ly8xZHJ2Lm1zL2IvcyFBbUg/root/content',
+          'https://api.onedrive.com/v1.0/shares/u!aHR0cHM6Ly8xZHJ2Lm1zL2IvaQ/root/content',
+        reason: 'removing the trailing `=` characters ',
```

Comment:
I'd suggest adding "in the base64 representation" to each of these reason strings, just to make it more obvious at a glance that this is not talking about a transformation of the input URL directly.